modules/snooker/src/main/ts/ephox/snooker/calc/Deltas.ts

```diff
@@ -28,7 +29,7 @@ const neighbours = function (input: number[], index: number) {
  * Calculate the offsets to apply to each column width (not the absolute widths themselves)
  * based on a resize at column: column of step: step. The minimum column width allowed is min
```

Comment:
Is the "min" part of this comment legacy? I assumed it was an arg at first

Reply:
Yeah, looks like it was originally an arg based on the git history. I'll remove that sentence since it is not relevant anymore.